src/v2/styles/TypingIndicator/TypingIndicator-theme.scss

```
@@ -0,0 +1,52 @@
+@use '../utils';
+
+.str-chat {
+  --str-chat__typing-indicator-font-family: var(--str-chat__font-family);
+  --str-chat__typing-indicator-border-radius: none;
+  --str-chat__typing-indicator-color: var(--str-chat__text-low-emphasis-color);
+  --str-chat__typing-indicator-background-color: var(--str-chat__overlay-color);
+  --str-chat__typing-indicator-border-block-start: none;
+  --str-chat__typing-indicator-border-block-end: none;
+  --str-chat__typing-indicator-border-inline-start: none;
+  --str-chat__typing-indicator-border-inline-end: none;
+  --str-chat__typing-indicator-box-shadow: none;
+  --str-chat__typing-indicator-dot-background-color: var(--str-chat__text-color);
```

```suggestion
  --str-chat__typing-indicator__dot-background-color: var(--str-chat__text-color);
```

Nitpick: I'm not sure how closely BEM is followed across other files but, I think `dot` represents an element. This just caught my eye :)

---

In theme-v2 for CSS variables we use camel-case after the `str-chat__` prefix

---

I think BEM is just for class names, but I'm happy to be corrected. Let's stick to snake-case for CSS variables.

---

Unfortunately, our CSS class names are not 100% consistent, it's mostly BEM, but there are exceptions, the only consistent part is the `str-chat__` prefix.

---

https://github.com/GetStream/stream-chat-css/issues/135 created an issue to create some lint rules to check class and variable names https://github.com/GetStream/stream-chat-css/issues/135